invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemMetadata/components/CreateNewLoanModal/CreateNewLoanModal.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { generatePath } from 'react-router';
+import { Button, Header, Icon, Modal } from 'semantic-ui-react';
+import { Loader } from '../../../../../../../common/components/Loader';
+import { BackOfficeURLS } from '../../../../../../../common/urls';
+import history from '../../../../../../../history';
+import _isEmpty from 'lodash/isEmpty';
+
+export default class CreateNewLoanModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+    this.createNewLoanForItem = this.props.createNewLoanForItem.bind(this);
+    this.onLoanCreatedCallback = this.props.onLoanCreatedCallback.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.resetNewLoanState();
+  }
+
+  close = () => {
+    this.setState({ open: false }, () => {
+      this.props.onLoanCreatedCallback(this.props.itemPid);
+    });
+  };
+
+  checkoutNewLoan = (itemPid, loan) => {
+    this.createNewLoanForItem(itemPid, loan);
+  };
+
+  triggerButton = (
+    <Button
+      positive
+      floated="right"
+      size="small"
+      onClick={() => this.setState({ open: true })}
+      disabled={this.props.disabled}
+    >
+      <Icon name="add" />
+      Create new loan
+    </Button>
+  );
+
+  gotoDetailsPage = loanPid => {
+    const path = generatePath(BackOfficeURLS.loanDetails, {
+      loanPid: loanPid,
+    });
+    history.push(path);
+  };
+
+  render() {
+    let { itemPid, isLoading, data, hasError } = this.props;
+    let loanData = {
+      item_pid: itemPid,
+      patron_pid: '111', // change this
```

Comment:
I would propose to have a common  'change me' label, so we can easily find these places later

Reply:
You are right I will change it with a `FIXME` :)